CHANGELOG.md

```diff
@@ -1,16 +1,42 @@
 Changelog
 =========
 
-## 0.6.0 SNAPSHOT (current master)
+## 0.6.0-RC1
+
+### breaking changes
 
 * reorganize maven packages: rename group parent to ohsome-parent (version bump to 2.3), rename local parent to oshdb-parent, and change groupId to org.heigit.ohsome
-* compatibility fix to allow building of javadoc under Java 11
-* fix bug where in some cases, instead of an OSHDBTimeoutException an IniteException was thrown. #258
+
+When switching to the OSHDB version 0.6 you need to adapt your `pom.xml` to the new groupId, e.g.:
```

Comment:
Should we mention that the `import`s in each file need to be changed?

Reply:
I would add that to the changelog with #241